static/src/javascripts/projects/commercial/modules/dfp/render-advert-label.js

```diff
@@ -39,3 +38,29 @@ export const renderAdvertLabel = (adSlotNode: HTMLElement): Promise<null> =>
             });
         }
     });
+
+export const renderStickyAdLabel = (adSlotNode: HTMLElement): Promise<null> =>
+    fastdom.write(() => {
+        const adSlotLabel = document.createElement('div');
+        adSlotLabel.classList.add('ad-slot__label');
+        adSlotLabel.classList.add('sticky');
+        adSlotLabel.innerHTML = 'Advertisement';
+        adSlotNode.appendChild(adSlotLabel);
+    });
+
+export const renderStickyScrollForMoreLabel = (
+    adSlotNode: HTMLElement
+): Promise<null> =>
+    fastdom.write(() => {
+        const scrollForMoreLabel = document.createElement('div');
+        scrollForMoreLabel.classList.add('ad-slot__scroll');
+        scrollForMoreLabel.innerHTML = 'Scroll for More';
+        scrollForMoreLabel.onclick = event => {
+            adSlotNode.scrollIntoView({
+                behavior: 'smooth',
+                block: 'end',
```

Comment:
Since this ad is for mobile are we polyfilling the scrollIntoView options?

Reply:
checked it in `caniuse.com` and it is supported by all major browsers. There are some (or older versions) which don't support the `smooth` option which is not a big issue as it will still perform the scroll functionality. I will assess it again in the future, as this 'Scroll for More' label are not yet confirm by UX team